changelog/v1.20.0-beta11/gloo-10950.yaml

```diff
@@ -0,0 +1,34 @@
+changelog:
+  - type: NEW_FEATURE
+    issueLink: https://github.com/solo-io/gloo/issues/10950
+    resolvesIssue: false
+    description: >-
+      Expose the max cache size for the OpenTelemetry collector in the tracing settings.
+  - type: DEPENDENCY_BUMP
+    dependencyOwner: envoyproxy v0.13.5-0.20250805143705-d51f8590a549
+    dependencyRepo: go-control-plane
+    dependencyTag: v0.13.5-0.20250805143705-d51f8590a549
+  - type: DEPENDENCY_BUMP
+    dependencyOwner: envoyproxy v0.13.5-0.20250805143705-d51f8590a549
+    dependencyRepo: go-control-plane/envoy
+    dependencyTag: v1.32.5-0.20250805143705-d51f8590a549
+  - type: DEPENDENCY_BUMP
+    dependencyOwner: opentelemetry v1.7.1
+    dependencyRepo: proto/otlp
+    dependencyTag: v1.7.1
+  - type: DEPENDENCY_BUMP
+    dependencyOwner: golang
+    dependencyRepo: crypto
+    dependencyTag: v0.40.0
+  - type: DEPENDENCY_BUMP
+    dependencyOwner: golang
+    dependencyRepo: sync
+    dependencyTag: v0.16.0
+  - type: DEPENDENCY_BUMP
+    dependencyOwner: golang
+    dependencyRepo: tools
+    dependencyTag: v0.34.0
+  - type: DEPENDENCY_BUMP
+    dependencyOwner: golang
+    dependencyRepo: grpc
+    dependencyTag: v1.74.2
```

Comment:
just out of curiosity, did you explicitly update these dependencies or were they pull in implicitly by the go-control-plane bump?